src/Features/VisualBasic/Portable/EditAndContinue/SyntaxComparer.vb

```diff
@@ -72,15 +72,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.EditAndContinue
 
             TypeParameterList                ' tied to parent
             TypeParameter                    ' tied to parent
-            TypeParameterConstraintClause    ' tied to parent
```

Comment:
>TypeParameterConstraintClause [](http://example.com/codeflow?start=12&length=29)

These nodes can be treated as parts of their parent nodes - we want to see updates for the parent nodes that are associated with symbols in semantic analysis.